Check/BestPractices/Fast404.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\BestPractices\Fast404.
+ */
+class SiteAuditCheckBestPracticesFast404 extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Fast 404 pages');
+  }
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Check if enabled.');
+  }
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {}
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultPass().
+   */
+  public function getResultPass() {
+    return dt('Fast 404 pages are enabled.');
+  }
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultWarn().
+   */
+  public function getResultWarn() {
+    return dt('Fast 404 pages are not enabled.');
+  }
+  /**
+   * Implements \SiteAudit\Check\Abstract\getAction().
+   */
+  public function getAction() {
+    if ($this->score == SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_WARN) {
+      return dt('See https://drupal.org/node/1313592 and default.settings.php for details on how to implement.');
```

Comment:
Sorry for the delay in response, I've been traveling (and still am).

https://github.com/drupal/drupal/blob/8.0.0-beta7/sites/default/default.settings.php#L572 actually has some pretty detailed instructions, linking to that line would be best. We need to make sure that we're not linking to unrelated instructions.